servicebus/data-plane/.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,52 @@
+﻿# Contribute or Provide Feedback for Azure Service Bus
```

Comment:
Similarly, I feel like this should be at the root rather than a sub directory.

Reply:
Will update the changes in one of next PR.